src/wui/buildingwindow.h

```diff
@@ -86,6 +86,10 @@ struct BuildingWindow : public UI::UniqueWindow {
 	void act_enhance(Widelands::DescriptionIndex, bool is_csite);
 	void clicked_goto();
 	void act_mute(bool all);
+	virtual void clicked_watch() {
```

Comment:
Could this be written as virtual void clicked_watch() == 0 ?

Reply:
Then I get "cannot instantiate XXXWindow because function clicked_watch is pure virtual" for all subclasses of BuildingWindow except ProductionsiteWindow